script.module.future/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.future" name="future" version="0.16.0" provider-name="Ed Schofield">
+<addon id="script.module.future" name="future" version="0.16.0.1" provider-name="Ed Schofield">
```

Comment:
I am not sure either but https://github.com/xbmc/repo-plugins/pull/916 was similar and it was merged despite my comment. So I guess it is fine.